clients/drcachesim/CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ endif()
 
```

Comment:
Please include in the PR description the size comparison for lz4 vs zip.  IIRC lz4 size was pretty good despite being faster, from evaluating it for the raw files, but let's have some actual numbers, as for some users size matters more than time for the raw2trace step.

Reply:
> Derek, we performed some measurements from conversion of trace got from Firefox browser startup. These are the results:
> 
> gz	zip DEFAULT	zip BEST_SPEED	lz4 fast	lz4 CLEVEL_MIN	gz+patch	lz4 fast +patch	lz4 CLEVEL_MIN +patch
> debug	13m24.289s (942.9 MB)	17m22.780s (945.5 MB)	16m25.397s (1.3 GB)	11m36.813s (1.4 GB)	11m48.851s (834.2 MB)	6m15.137s (942.9 MB)	4m22.005s (1.4 GB)	4m47.105s (834.2 MB)
> release	3m52.183s (942.9 MB)	5m10.639s (945.5 MB)	4m3.868s (1.3 GB)	2m5.202s (1.4 GB)	2m26.787s (834.2 MB)	3m10.944s (942.9 MB)	1m9.434s (1.4 GB)	1m32.259s (834.2 MB)

thx Yury for data